hyperspace/solana/src/lib.rs

```diff
@@ -248,8 +248,8 @@ impl IbcProvider for SolanaClient {
 					let validator_idx = all_validators
 						.iter()
 						.position(|v| {
-							v.pubkey
-								== PubKey::from_bytes(&validator.to_bytes().as_slice()).unwrap()
+							v.pubkey ==
+								PubKey::from_bytes(&validator.to_bytes().as_slice()).unwrap()
```

Comment:
I realise this is just formatting change so not really applicable to
this PR, but by the way, this probably could be simplified to

```suggestion
							v.pubkey.to_bytes() == validator.to_bytes()
```

Reply:
Fixed